snap/snapcraft.yaml

```
@@ -1,8 +1,17 @@
 name: sublime-text
 adopt-info: sublime-text
 summary: A sophisticated text editor for code, markup and prose.
+title: Sublime Text
 description: |
-  A sophisticated text editor for code, markup and prose.
+  Sublime Text is a powerful multi-platform text editor with a clean, modern user interface.
+  Bringing wide support for programming languages, extensions, syntax-based features and many other useful features alongside low resource usage, Sublime Text proves itself to be a very competent solution for a vast set of environments and applications.
+  
+  Note - Sublime Text may be downloaded and evaluated for free, however a license must be purchased for continued use.
```

Best to also add a line explaining this is an unofficial package.

```suggestion
  Note - Sublime Text may be downloaded and evaluated for free, however a license must be purchased for continued use.
  
  **Authors**
  
  This snap is maintained by the Snapcrafters community, and is not necessarily endorsed or officially maintained by the upstream developers.
```